qfieldsync/core/cloud_api.py

```diff
@@ -182,19 +260,36 @@ def server_urls() -> List[str]:
             "https://localhost:8002/",
         ]
 
+    def username(self) -> Optional[str]:
```

Comment:
all the methods we have in the codebase are having some kind of a verb, so `get_username` is a good candidate. Another option would be to convert it to dynamic `@property`. But don't mix the two.